misc/license_headers.py

```diff
@@ -9,40 +9,9 @@
 from itertools import chain
 from typing import Iterable, Iterator
 
-THIS_YEAR = "2021"
-BASE_HEADER = (
-    f"Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2016-{THIS_YEAR} Scille SAS"
-)
-HEADERS = {".py": f"# {BASE_HEADER}", ".sql": f"-- {BASE_HEADER}"}
-HEADER_RE = re.compile(
-    r"^(#|--) Parsec Cloud \(https://parsec\.cloud\) Copyright \(c\) AGPLv3 2016-(?P<year>[0-9]{4}) Scille SAS$"
-)
-assert all(HEADER_RE.match(x) for x in HEADERS.values())  # Sanity check
 
-SKIP_PATHES = (Path("parsec/core/gui/_resources_rc.py"), Path("parsec/core/gui/ui/"))
-
-
-def need_skip(path: Path) -> bool:
-    for skip_path in SKIP_PATHES:
-        try:
-            path.relative_to(skip_path)
-            return True
-        except ValueError:
-            pass
-    return False
-
-
-def get_files(pathes: Iterable[Path]) -> Iterator[Path]:
-    for path in pathes:
-        if path.is_dir():
-            for f in chain(Path(path).glob("**/*.py"), Path(path).glob("**/*.sql")):
-                if need_skip(f):
-                    continue
-                yield f
-        elif path.is_file():
-            yield path
-        elif not path.exists():
-            raise SystemExit(f"Error: Path `{path}` doesn't exist !")
+THIS_YEAR = "2021"  # Change me when new year comes ;-)
```

Comment:
Why not use `date.today()` like below?

Reply:
Because this would cause a CI breakage every January 1rst which requires modifying each file to solve :'(
So I think it's better to manually control when the change should occur